core/templates/components/common-layout-directives/common-elements/classroom-navigation-links.component.html

```diff
@@ -0,0 +1,185 @@
+<div class="classroom-navigation-links-container">
+  <ng-container *ngIf="isLoading; else content">
+    <div class="skeleton-loader"></div>
+  </ng-container>
+  <ng-template #content>
+    <div class="classroom-navigation-links-content">
+      <div class="multiple-classroom-nav-links" *ngIf="classroomSummaries.length > 0">
+        <div class="multiple-classrooms-intro">
+          <a href="/learn">{{ 'I18N_CLASSROOM_NAVIGATION_LINKS_INTRO' | translate }}</a>
+          <p>{{ 'I18N_CLASSROOM_NAVIGATION_LINKS_DESCRIPTION' | translate }}</p>
+        </div>
+        <div *ngFor="let classroom of classroomSummaries | slice:0:2" class="classroom-nav-item-container">
```

Comment:
I would just store two classrooms instead of slicing it here. There's not much sense to store all classrooms since we show only the first two. Also, it'd be easier to test

Reply:
Yeah make sense. Done.